test/suite/test_inmem01.py

```diff
@@ -88,6 +88,7 @@ def test_insert_over_delete(self):
 
     # Run queries after adding, removing and re-inserting data.
     # Try out keeping a cursor open while adding new data.
+    @unittest.skip("Temporarily disabled")
```

Comment:
This is a known issue with the current `develop` branch. I'm just disabling this here since this test failure is generating noise and hiding legitimate failures.